pkg/webhook/admission/pod/metrics_aggregate_injector.go

```diff
@@ -108,7 +109,7 @@ func (ma *MetricsAggregator) InjectMetricsAggregator(pod *v1.Pod) error {
 		// If enableMetricAggregation is true, set it as the queue proxy metrics aggregation port.
 		podPromPort := constants.DefaultPodPrometheusPort
 		if enableMetricAggregation == "true" {
-			podPromPort = strconv.Itoa(constants.QueueProxyAggregatePrometheusMetricsPort)
+			podPromPort = strconv.Itoa(int(constants.QueueProxyAggregatePrometheusMetricsPort))
```

Comment:
isn't better to keep the field as string?

Reply:
As explained above, it will lead to downcasting.